tests/benchmarks/multicore/idle_stm/testcase.yaml

```diff
@@ -0,0 +1,52 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore
+  platform_allow:
+    - nrf54h20dk/nrf54h20/cpuapp
+  integration_platforms:
+    - nrf54h20dk/nrf54h20/cpuapp
+
+tests:
+  benchmarks.multicore.idle_stm.nrf54h20dk_cpuapp_cpurad.no_sleep:
+    required_snippets:
+      - nordic-log-stm
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Multicore idle_stm test on"
+        - "rad/idle_stm: Run"
+        - "ppr/idle_stm: Run"
+        - "app/idle_stm: Run"
+
+  benchmarks.multicore.idle_stm.nrf54h20dk_cpuapp_cpurad.idle:
+    tags: ppk_power_measure
+    required_snippets:
+      - nordic-log-stm
+    extra_args:
+      idle_stm_CONF_FILE=prj_s2ram.conf
+      remote_rad_CONF_FILE=prj_s2ram.conf
+      remote_ppr_CONF_FILE=prj_s2ram.conf
+      idle_stm_CONFIG_TEST_SLEEP_DURATION_MS=500
+      remote_rad_CONFIG_TEST_SLEEP_DURATION_MS=500
+      remote_ppr_CONFIG_TEST_SLEEP_DURATION_MS=500
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_pwm_and_idle"
```

Comment:
pwm?

Reply:
Current measurements not needed as SOC can't enter deepest low power states due to use of STM.